File: 프로그래머스/lv0/120811. 중앙값 구하기/중앙값 구하기.java
import java.util.Arrays;

class Solution {
    public int solution(int[] array) {
        int answer = 0;
        Arrays.sort(array);
        // 배열 오름차순 정리
        answer = array[array.length / 2];
        // 배열의 길이를 2로 나누어 중앙값을 찾는다
        return answer;
    }
}

File: 프로그래머스/unrated/120811. 중앙값 구하기/중앙값 구하기.js
function solution(array) {
    var answer = 0;
    array.sort((a, b) => a - b);
    // 배열 오름차순 정렬
    answer = array[Math.floor(array.length / 2)];
    // 배열의 길이를 2로 나누어 중앙값을 찾는다
    return answer;
}
